src/app/core-ui/main/main-view.component.ts

```diff
@@ -62,7 +62,10 @@ export class MainViewComponent implements OnInit {
     // Updates the error box in the sidenav whenever a stateCall returns an error.
     this._rpc.errorsStateCall.asObservable()
     .subscribe(status => this.daemonRunning = true,
```

Comment:
```
this.rpc.state.observe('blocks').throttle(val => Observable.interval(30000/*ms*/)).subscribe(block =>  this.rpc_update());
```
Use a throttle here

Reply:
yup, but not sure why you did polling for blocks as it refresh the whole table and I saw you removed my previous fix but that fix to prevent the whole table to refresh. 
@allienworks  we have discussion on this I guess ?